assets/scss/_call-to-action.scss

```
@@ -1,17 +1,8 @@
 .call-to-action {
   position: relative;
   background-color: $gray-800;
-  background: url('../img/bg-masthead.jpg') no-repeat center center;
+  background: url('../img/roll-coupons.jpg') no-repeat center center;
   @include background-cover;
   padding-top: 5rem;
   padding-bottom: 5rem;
-  .overlay {
-    position: absolute;
-    background-color: $gray-900;
-    height: 100%;
-    width: 100%;
-    top: 0;
-    left: 0;
-    opacity: 0.3;
-  }
 }
```

Que penses tu d'ajouter un petit overlay blanc pour rendre l'image un peu moins contrasté?

Par exemple :
```suggestion
  &:after {
      position: absolute;
      content: '';
      display: block;
      top: 0;
      left: 0;
      right: 0;
      bottom: 0;
      background: rgba(255, 255, 255, 0.2);
      pointer-events: none;
  }
}
```

---

Au top ! Je m'étais fait la même remarque sans pour autant arrivé à quelque chose qui me plaît, du coup j'avais abandonné l'idée. Merci